tests/foreman/cli/test_reporttemplates.py

```diff
@@ -128,15 +128,15 @@ def test_positive_report_help(self):
         :CaseImportance: High
         """
         base = Base().execute('--help')
-        self.assertGreater(len([i for i in base if 'report-template' in i]), 0)
+        assert len([i for i in base if 'report-template' in i]) > 0
```

Comment:
`0` is falsy, so this doesn't even need the >0. `len` will always be positive (or zero).

`assert len([])` raises AssertionError